### ISSUE TITLE
Subtitles for the hearing impaired (that are not technically closed cpations)

### ISSUE THREAD

**Issue by JeannetteHo1** [w3c/a11y-discov-vocab#104]
I noticed that the Adaption Term "captions" has been deprecated and replaced by "closedCaptions" and "openCaptions" for the accessibilityFeature property.

I was wondering if we could use those terms with videos that contain subtitles for the hearing impaired (SDH).  They serve the same function as closed captions, with descriptions of the settings, nonverbal sounds, etc. in addition to the dialogue of a film. However, they are not the same thing from a technical standpoint.  The way they appear and the way they are technically encoded and transmitted differ from traditional closed captions. For instance, they appear like regular subtitles rather than white letters against a black background, even though their content is the same as what you would see in traditional closed captioning.

Sometimes they can be "turned off" and other times not.  Would it be appropriate to apply "closedCaptions" and "openCaptions" to them as well? It is not clear from the way the documentation is written, and since cataloging librarians are starting to apply this vocabulary to describe videos with such features, I think this should be clarified.

**Comment by madeleinerothberg** [comment]
Hi Jeannette,

Yes, you can use the “captions” properties for subtitles for the deaf and hard of hearing. You would use the open and closed versions of the metadata to match whether the SDH can be turned off (closed) or are always visible (open). Functionally it is the same, though as you said the technology may be a bit different.

Best,
Madeleine

Madeleine Rothberg
Senior Subject Matter Expert
617-300-2492


From: JeannetteHo1 ***@***.***>
Date: Thursday, February 1, 2024 at 5:59 PM
To: w3c/a11y-discov-vocab ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [w3c/a11y-discov-vocab] Subtitles for the hearing impaired (that are not technically closed cpations) (Issue #104)

I noticed that the Adaption Term "captions" has been deprecated and replaced by "closedCaptions" and "openCaptions" for the accessibilityFeature property.

I was wondering if we could use those terms with videos that contain subtitles for the hearing impaired (SDH). They serve the same function as closed captions, with descriptions of the settings, nonverbal sounds, etc. in addition to the dialogue of a film. However, they the same thing from a technical standpoint. The way they appear and the way they are technically encoded and transmitted differ from traditional closed captions. For instance, they appear like regular subtitles rather than white letters against a black box, even though their content is the same as what you would see in traditional closed captioning.

Sometimes they can be "turned off" and other times not. Would it be appropriate to apply "closedCaptions" and "openCaptions" to them as well?

**Comment by JeannetteHo1** [comment]
Thanks for letting me know. The question was indeed answered.

Jeannette

Jeannette Ho
Cataloging/Metadata Librarian
Texas A&M University Libraries
________________________________
